packages/web-api/src/WebClient.ts

```diff
@@ -885,3 +891,44 @@ function warnIfThreadTsIsNotString(method: string, logger: Logger, options?: Web
 export function buildThreadTsWarningMessage(method: string): string { 
   return `The given thread_ts value in the request payload for a ${method} call is a float value. We highly recommend using a string value instead.`;
 }
+
+/**
+ * Takes an object and redacts specific items
+ * @param body 
+ * @returns 
```

Comment:
looks like the variable(s) returned in this function might needed to be added here!

Reply:
Just going to remove the jsdoc annotation!